extension/src/cli/dvc/version.test.ts

```diff
@@ -57,125 +56,114 @@ describe('isVersionCompatible', () => {
     patch: latestTestedPatch
   } = extractSemver(LATEST_TESTED_CLI_VERSION) as ParsedSemver
 
-  it('should be compatible and not send a toast message if the provided version matches the min version', () => {
```

Comment:
[F] The message has been hoisted out of this function so that it can be changed for different situations. The tests have been moved to `extension/setup.test.ts`